systems/ukama-agent/cdr/pkg/server/cdr.go

```diff
@@ -75,8 +81,24 @@ func (s *CDRServer) InitUsage(imsi string, policy string) error {
 	err = s.usageRepo.Add(&u)
 	if err != nil {
 		log.Errorf("Error initalizing usage for imsi %s. Error %+v", imsi, err)
+
 		return err
 	}
+
+	asr, err := s.asrClient.GetAsr(imsi)
+	if err == nil && asr.Record != nil && asr.Record.Policy != nil && asr.Record.Policy.Uuid == policy {
+		labels := map[string]string{
+			// "package":  asr.Record.SimPackageId,
+			"dataplan": asr.Record.PackageId,
+			// "network":  asr.Record.Network,
```

Comment:
Are we not setting sim package Id and network id labels?

Reply:
In this branch, these fields do not yet exist in the ASR service. The set of updates that enable those are in a different PR. So at this point, I had 2 choices: either 1) I duplicate the same changes for the other PR here too or 2) I just marked them as comment, and when I merge the other PR which is also in this milestone, I'll un-comment these line before merge into main.  I choose option 2) which way easier and less error prone :smile: